### PR TITLE
PP 7118 remove 3ds in cookie validation

### PR DESCRIPTION
**Pull request by narinderchana**
Description:
- Remove session cookie validation for /card_details/:chargeId/3ds_required_in & /card_details/:chargeId/3ds_required_in requests

### CODE REVIEW

File: test/integration/three_d_secure_ft_tests.js [hunk 1]
```diff
@@ -60,6 +60,8 @@ describe('chargeTests', function () {
         getChargeRequest(app, cookieValue, chargeId, '/3ds_required_out')
           .expect(200)
           .expect(function (res) {
+            expect(res.header['set-cookie'][0].startsWith('frontend_state=')).to.be.true
+            expect(res.header['set-cookie'][0].length).to.be.greaterThan(100)
```
Comment (alexbishop1) [inline test/integration/three_d_secure_ft_tests.js:64]: I think we decided that it wasn’t worth checking we’re setting the cookie here. We just want to check on the 3DS required in pages that we definitely don’t.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks good: just one thing about the tests.

File: test/integration/three_d_secure_ft_tests.js (hunk 1 above)
Comment (narinderchana) [reply test/integration/three_d_secure_ft_tests.js]: This check was in a few places, not only lines 63/64. I have removed the other reference too.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks good.

To assist our future selves, could you edit the commit message to explain why we’re doing this? Something like this: “Users arrive at the 3DS required in page via a cross-site request and modern browsers treat cookies as SameSite=lax by default, which means we won’t sometimes won’t get sent the cookie when it’s a POST request. We’ll still require the cookie to continue further on the journey and actually process the 3DS result.”

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**